Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1110,7 +1110,7 @@ void RenderGrid::placeSpecifiedMajorAxisItemsOnGrid(const Vector<RenderBox*>& au
 
     // Mapping between the major axis tracks (rows or columns) and the last auto-placed item's position inserted on
     // that track. This is needed to implement "sparse" packing for items locked to a given track.
-    // See http://dev.w3.org/csswg/css-grid/#auto-placement-algorithm
+    // See https://dev.w3.org/csswg/css-grid/#auto-placement-algo
```

Comment:
```suggestion
    // See https://drafts.csswg.org/css-grid/#auto-placement-algo
```
I'd use this link, but no strong opinions.

Reply:
Yeah, that is probably a better link. Let me fix that.